src/main/java/org/opentripplanner/apis/gtfs/GtfsGraphQLIndex.java

```diff
@@ -118,6 +111,7 @@ protected static GraphQLSchema buildSchema() {
         .scalar(GraphQLScalars.COORDINATE_VALUE_SCALAR)
         .scalar(GraphQLScalars.COST_SCALAR)
         .scalar(GraphQLScalars.RELUCTANCE_SCALAR)
+        .scalar(GraphQLScalars.LOCAL_DATE_SCALAR)
```

Comment:
Btw what is the difference between our own LOCAL_DATE_SCALAR vs ExtendedScalars.Date?

Reply:
There is quite an overlap but it seems that `ExtendedScalars.Date` allows a bit wider range of Java types to be serialised into scalar values.

Reply:
We will keep ours.